interfaces/builtin/repowerd.go

```diff
@@ -0,0 +1,307 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) 2016 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package builtin
+
+import (
+	"bytes"
+	"fmt"
+
+	"github.com/snapcore/snapd/interfaces"
+	"github.com/snapcore/snapd/release"
+)
+
+var repowerdPermanentSlotAppArmor = []byte(`
+# Description: Allow operating as the repowerd service. Reserved because this
+#  gives privileged access to the system.
+# Usage: reserved
+
+# HW alarms
+/dev/alarm rw,
+
+# Suspend
+/sys/power/** rw,
+
+# Backlight
+/sys/**/brightness rw,
+/sys/**/max_brightness r,
+/sys/**/type r,
+
+# DBus
+#include <abstractions/dbus-strict>
+
+dbus (send)
+  bus=system
+  path=/org/freedesktop/DBus
+  interface=org.freedesktop.DBus
+  member={GetId,RequestName,ReleaseName,GetConnectionUnixProcessID}
+  peer=(name=org.freedesktop.DBus),
+
+dbus (receive, send)
+  bus=system
+  interface=org.freedesktop.DBus.{Properties,Introspectable},
+
+dbus (bind)
+  bus=system
+  name=com.canonical.powerd,
+
+dbus (bind)
+  bus=system
+  name=com.canonical.Unity.Screen,
```

Comment:
repowerd binds to two different well-known names? Should this be two different interfaces and repowerd slots both? Why is the well-known name com.canonical.powerd instead of com.canonical.repowerd? Should the interface be name 'powerd' instead?